### PR TITLE
explanation of MySQL Pod startupProbe warning

### PR DESCRIPTION
**Pull request by fansi404**
1. rename `系统故障 FAQ` to `部署运维 FAQ`
2. remove `账号登录不了，报 504` from `系统故障 FAQ`, move it to `诊断手册`
3. add explanation of MySQL Pod startupProbe warning

Signed-off-by: fansi <fansiqiong@koderover.com>

---
This change is [Reviewable](https://reviewable.io/reviews/koderover/zadig-doc/148)

### CODE REVIEW

**Comment by fansi404** [comment]
request for review. @lilianzhu @flyer103

File: zh_cn/13.常见问题/01.debug-system-faq.md [hunk 1]
```diff
@@ -1,25 +1,9 @@
 ---
-title: 系统故障 FAQ
+title: 部署运维 FAQ
 date: 2021-03-31 09:32:30
```
Comment (flyer103) [inline zh_cn/13.常见问题/01.debug-system-faq.md:3]: Does the date need to be updated?

File: zh_cn/13.常见问题/3.troubleshooting-guide.md [hunk 2]
```diff
@@ -3,6 +3,23 @@ title: 诊断手册
 date: 2021-03-31 14:16:53
 permalink: /faq/troubleshooting-guide
 ---
+
+## 账号登录不了，报 504
+
+### Zadig 系统账号无法登录
+
+查看当前 pd-web 服务的状态和日志，确认和数据库服务连接是否存在异常。命令如下：
```
Comment (jamsman94) [inline zh_cn/13.常见问题/3.troubleshooting-guide.md:11]: Does not apply anymore

**Review by jamsman94 — CHANGES_REQUESTED** [review]

File: zh_cn/13.常见问题/01.debug-system-faq.md (hunk 1 above)
Comment (fansi404) [reply zh_cn/13.常见问题/01.debug-system-faq.md:3]: It is just some information of when the file was created(generated automaticly). used nowhere for now.

**Review by flyer103 — APPROVED** [review]

**Review by jamsman94 — APPROVED** [review]
LGTM

**Pull request merged**